sixtyfps_compiler/passes/default_geometry.rs

```diff
@@ -125,6 +125,9 @@ fn gen_layout_info_prop(elem: &ElementRc) {
         .borrow()
         .children
         .iter()
+        .filter(|c| {
+            !c.borrow().bindings.contains_key("x") && !c.borrow().bindings.contains_key("y")
```

Comment:
Does this filter out "manually" placed/bound children, right? Might be worth a comment